src/main/java/seedu/address/model/person/information/RiskLevel.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.model.person.information;
+
+import static java.util.Objects.requireNonNull;
+import java.util.HashSet;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.address.model.person.information.RiskLevel.Risk.LOW;
+
+public class RiskLevel {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Risk level should only contain 3 types of values, low, medium or high";
+
+    public final Risk riskStatus;
+
+    public enum Risk {
+        LOW,
+        MEDIUM,
+        HIGH;
+    }
+
+    public RiskLevel(Risk risk) {
+        requireNonNull(risk);
+        this.riskStatus = risk;
+    }
+
+    public RiskLevel(String risk) {
+        requireNonNull(risk);
+        this.riskStatus = Risk.valueOf(risk);
+    }
+
+    public static boolean isValidRisk(String risk) {
+        HashSet<String> set = new HashSet<>();
+        for (Risk riskSet: Risk.values()) {
+            set.add(riskSet.name());
+        }
+        return set.contains(risk.toUpperCase());
+    }
+
+    @Override
+    public String toString() {
+        switch (riskStatus) {
+        case LOW:
+            return "low";
+        case MEDIUM:
+            return "medium";
+        case HIGH:
+            return "high";
+        default:
+            return "invalid";
+        }
+
+    }
```

Comment:
Perhaps we can use riskStatus.toString().toLowerCase();
```suggestion
    public String toString() {
        return riskStatus.toString().toLowerCase();
    }
```

Reply:
Another option would be to use name()
```
public String toString() {
    return riskStatus.name().toLowerCase();
}
```
The only difference between name() and toString() is that toString() can be overridden.